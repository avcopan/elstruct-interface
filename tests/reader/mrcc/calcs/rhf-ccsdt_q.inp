calc=CCSDT(Q)
ccprog=mrcc
scftype=RHF
basis=6-31G*
charge=0
mult=1

geom=xyz
3
 
O          0.0000000000       -0.0530229169        0.0000000000
H          0.0000000000        0.5556614584       -0.7504910643
H          0.0000000000        0.5556614584        0.7504910643

