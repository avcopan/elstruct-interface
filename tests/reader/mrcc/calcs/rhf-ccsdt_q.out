 **********************************************************************
                          MRCC program system
 **********************************************************************
 
                              Written by
  Mihaly Kallay, Zoltan Rolik, Jozsef Csontos, Peter Nagy, Gyula Samu,
      David Mester, Jozsef Csoka, Bernat Szabo, Istvan Ladjanszki,
       Lorant Szegedy, Bence Ladoczki, Klara Petrov, Mate Farkas,
                    Pal D. Mezei, and Bence Hegely
 
        Department of Physical Chemistry and Materials Science
            Budapest University of Technology and Economics
                  Budapest P.O.Box 91, H-1521 Hungary
 
                              www.mrcc.hu
 
                       Release date: May 23, 2018

 ************************ 2019-01-13 18:35:22 *************************
 Executing minp...

 Reading input from MINP...
 
 Input file:
 
calc=CCSDT(Q)
ccprog=mrcc
scftype=RHF
basis=6-31G*
charge=0
mult=1

geom=xyz
3
 
O          0.0000000000       -0.0530229169        0.0000000000
H          0.0000000000        0.5556614584       -0.7504910643
H          0.0000000000        0.5556614584        0.7504910643

 
 
 Checking keyword combinations...
 
 Keywords:
 
 active=none
 agrid=ld0006-ld0590
 basis=6-31g*
 basis_sm=none
 basopt=off
 bfbasis=none
 bpcompo=0.985
 bpcompv=0.98
 bpdfo=0.985
 bpocc=0.985
 bppdo=0.985
 bppdv=0.98
 bpedo=0.985
 bpedv=0.98
 calc=ccsdt(q)
 ccmaxit=50
 ccprog=mrcc
 cctol=6
 charge=0
 cialg=disk
 ciguess=off
 cmpgrp=auto
 core=frozen
 corembed=off
 dboc=off
 dendec=cholesky
 dens=0
 dfalg=lineq
 dfbasis_cor=none
 dfbasis_scf=none
 dfintran=ovirt
 dft=off
 diag=david
 domrad=10.d0
 drpaalg=fit
 dual=off
 ecp=auto
 edisp=off
 embed=off
 epert=none
 eps=0.975
 excrad=0.d0
 excrad_fin=0.000000000D+00
 freq=off
 gauss=spher
 geom=xyz
 gopt=off
 ghost=none
 gtol=7
 grdens=off
 grtol=10
 hamilton=dc
 iface=none
 intalg=auto
 itol=10
 laptol=1.d-2
 lcorthr=loose
 lccrest=off
 lmp2dens=on
 lnoepso=0.d0
 lnoepsv=1e-6
 localcc=off
 localcorrsymm=off
 locintrf=disk
 maxact=off
 maxdim=100
 maxex=0
 mem=256MB
 molden=on
 mulmet=0
 mult=1
 nacto=0
 nactv=0
 naf_cor=off
 naf_scf=off
 nafalg=albe
 naftyp=
 nchol=auto
 ndeps=1e-3
 nstate=1
 nsing=0
 ntrip=0
 occ=
 optalg=
 optmaxit=50
 optetol=1e-6
 optgtol=1e-4
 optstol=1e-3
 orblocc=off
 orbloco=off
 orblocv=off
 orblocguess=cholesky
 osveps=1e-3
 ovirt=off
 ovltol=1e-7
 ovosnorb=80.0
 ptfreq=0.0
 popul=off
 pressure=100000
 qscf=off
 qmmm=off
 redcost_exc=off
 refdet=none
 rest=0
 rgrid=log3
 rohftype=semicanonical
 scfalg=auto
 scfdamp=off
 scfdiis=on
 scfdiis_end=50
 scfdiis_start=1
 scfdiis_step=1
 scfdtol=7
 scfext=10
 scfiguess=sad
 scflshift=off
 scfmaxit=50
 scftype=rhf
 scftol=6
 scspe=1.d0
 scsps=1.2d0
 scspt=0.33333333333333
 scspv=1.d0
 spairtol=1e-4
 symm=
 talg=occ
 temp=298.15
 test=off
 tprint=off
 uncontract=off
 unit=angs
 verbosity=2
 wpairtol=0.100000000E-05

 ************************ 2019-01-13 18:35:22 *************************
 Executing integ...

 Allocation of 256.0 Mbytes of memory...
 
 
 Number of atoms:              3
 Charge:                       0
 Number of electrons:         10
 Number of core electrons:     2
 Spin multiplicity:            1
 
                 Cartesian coordinates [bohr]      Atomic mass [AMU]  Z
   1  O     0.00000000   -0.10019879    0.00000000    15.994915       8
   2  H     0.00000000    1.05004797   -1.41822257     1.007825       1
   3  H     0.00000000    1.05004797    1.41822257     1.007825       1
 
 This molecule is an asymmetric top.
 This molecule is planar.
 
 Rotational constants [cm-1]: 
     25.418063211692     14.848764111928      9.373145338200
 
 Point group: C2v 
 Computational point group: C2v 
 
 Cartesian coordinates in standard orientation [bohr]
   1  O     0.00000000    0.00000000   -0.12872972
   2  H     0.00000000   -1.41822257    1.02151704
   3  H     0.00000000    1.41822257    1.02151704
 
 Nuclear repulsion energy [au]:          9.114682990675
 
 
 Basis set information:
 
 Maximum angular momentum: d
 Maximum number of contracted Gaussians:    3
 Maximum number of primitive Gaussians:    10
 Spherical harmonic GTOs are used.
 
    1  O  6-31g* [ 10s 4p 1d | 3s 2p 1d ]
    2  H  6-31g [ 4s | 2s ]
    3  H  6-31g [ 4s | 2s ]
 
 Total number of basis functions:     18
 
 
 Generating initial guess for the SCF calculation...
 CPU time [min]:     0.002                   Wall time [min]:     0.016
 
 Character table for point group C2v:
 
          E  C2  Qh  Qv  
    A1    1   1   1   1
    B1    1  -1   1  -1
    B2    1  -1  -1   1
    A2    1   1  -1  -1
 
 Number of basis functions per irrep:
    A1      9
    B1      5
    B2      3
    A2      1
 
 
 Calculation of overlap integrals...
 CPU time [min]:     0.002                   Wall time [min]:     0.016
 
 Calculation of the square root of the overlap matrix...
 Minimum eigenvalue of the overlap matrix: 0.632076E-01
 CPU time [min]:     0.002                   Wall time [min]:     0.016
 
 Calculation of kinetic energy integrals...
 CPU time [min]:     0.002                   Wall time [min]:     0.016
 Calculation of nuclear attraction integrals...
 CPU time [min]:     0.002                   Wall time [min]:     0.016
 
 Calculation of prescreening integrals...
 CPU time [min]:     0.002                   Wall time [min]:     0.016
 
 Calculation of two-electron integrals...
   7% done.
  38% done.
  63% done.
 100% done.
 CPU time [min]:     0.003                   Wall time [min]:     0.017

 ************************ 2019-01-13 18:35:23 *************************
 Executing scf...

 Allocation of 256.0 Mbytes of memory...
 ======================================================================
 ITERATION STEP   1
 CPU time [min]:     0.001                   Wall time [min]:     0.001
 
 ALPHA OCC:   3   1   1   0
 BETA  OCC:   3   1   1   0
 ***HARTREE-FOCK ENERGY IN STEP   1 IS     -75.7244350893339515 [AU]
 ======================================================================
 ITERATION STEP   2
 CPU time [min]:     0.002                   Wall time [min]:     0.002
 
 ALPHA OCC:   3   1   1   0
 BETA  OCC:   3   1   1   0
 ***HARTREE-FOCK ENERGY IN STEP   2 IS     -75.9945408969137333 [AU]
 ======================================================================
 ITERATION STEP   3
 CPU time [min]:     0.002                   Wall time [min]:     0.002
 
 ALPHA OCC:   3   1   1   0
 BETA  OCC:   3   1   1   0
 ***HARTREE-FOCK ENERGY IN STEP   3 IS     -76.0056286409736686 [AU]
 ======================================================================
 ITERATION STEP   4
 CPU time [min]:     0.002                   Wall time [min]:     0.002
 
 ALPHA OCC:   3   1   1   0
 BETA  OCC:   3   1   1   0
 ***HARTREE-FOCK ENERGY IN STEP   4 IS     -76.0082060351587927 [AU]
 ======================================================================
 ITERATION STEP   5
 CPU time [min]:     0.002                   Wall time [min]:     0.002
 
 ALPHA OCC:   3   1   1   0
 BETA  OCC:   3   1   1   0
 ***HARTREE-FOCK ENERGY IN STEP   5 IS     -76.0083481290693328 [AU]
 ======================================================================
 ITERATION STEP   6
 CPU time [min]:     0.002                   Wall time [min]:     0.002
 
 ALPHA OCC:   3   1   1   0
 BETA  OCC:   3   1   1   0
 ***HARTREE-FOCK ENERGY IN STEP   6 IS     -76.0083503450988331 [AU]
 ======================================================================
 ITERATION STEP   7
 CPU time [min]:     0.002                   Wall time [min]:     0.002
 
 ALPHA OCC:   3   1   1   0
 BETA  OCC:   3   1   1   0
 ***HARTREE-FOCK ENERGY IN STEP   7 IS     -76.0083504154681577 [AU]
 ======================================================================
 ITERATION STEP   8
 CPU time [min]:     0.003                   Wall time [min]:     0.003
 
 ALPHA OCC:   3   1   1   0
 BETA  OCC:   3   1   1   0
 ***HARTREE-FOCK ENERGY IN STEP   8 IS     -76.0083504222043587 [AU]
 ======================================================================
 ITERATION STEP   9
 CPU time [min]:     0.003                   Wall time [min]:     0.003
 
 ALPHA OCC:   3   1   1   0
 BETA  OCC:   3   1   1   0
 ***HARTREE-FOCK ENERGY IN STEP   9 IS     -76.0083504223749742 [AU]
 ======================================================================
 ITERATION STEP  10
 CPU time [min]:     0.003                   Wall time [min]:     0.003
 
 ALPHA OCC:   3   1   1   0
 BETA  OCC:   3   1   1   0
 ***HARTREE-FOCK ENERGY IN STEP  10 IS     -76.0083504223784416 [AU]
 ======================================================================
 
 SUCCESS...
 THE SCF ITERATION HAS CONVERGED!
 
                   A1  B1  B2  A2  
 FINAL ALPHA OCC:   3   1   1   0
 FINAL BETA  OCC:   3   1   1   0
 
 ***FINAL HARTREE-FOCK ENERGY:    -76.0083504223784416 [AU]
 
 RETURNING FROM SCF ALGORITHM
 ======================================================================

 ************************ 2019-01-13 18:35:23 *************************
 Executing ovirt...

 ovirt, the routine of orbital optimization and integral transformation 
Sun Jan 13 18:35:23 CST 2019
 Allocated memory:                     256  Mb
 RHF calculation! 
 integral transforamtion: AOs ------ MOs
 # of basis functions, # of int. blocks    18     1
100 % 
 second part
100 %
 integral transformation is completed! 
CPU and WC time           0.01           0.01     AOs -> MOs 
 inttol =   1.000000000000000E-010   10.0000000000000     
Integrals are written to fort.55
Sun Jan 13 18:35:23 CST 2019
  ovirt terminated normally 

 ************************ 2019-01-13 18:35:23 *************************
 Executing goldstone...

 Generation of CC equations in terms of H and T...
 Generation of antisymmetrized Goldstone diagrams...
 Number of diagrams in T^1 equations:    15
 Number of diagrams in T^2 equations:    38
 Number of diagrams in T^3 equations:    49
 Number of diagrams in T^4 equations:     5
 Translation of diagrams to factorized equations...
 Optimizing intermediate calculation...
 Number of floating-point operations per iteration step:   1.3506E+08
 Probable CPU time per iteration step (hours):      0.00
 Required memory (Mbytes):       4.1
 Number of intermediates:                                  80
 Number of intermediates to be stored:                     31
 Length of intermediate file (Mbytes):       0.8

 ************************ 2019-01-13 18:35:23 *************************
 Executing xmrcc...

 **********************************************************************
 CC(3)(4) calculation 
 
 
 Allocation of 100.0 Mbytes of memory...
 Number of spinorbitals:                    34
 Number of alpha electrons:                        4
 Number of beta electrons:                         4
 Spin multiplicity:                     1
 z-component of spin:  0.0
 Spatial symmetry:                      1
 Convergence criterion:  1.0E-06
 Construction of occupation graphs...
 Number of                     0 -fold excitations:                      1
 Number of                     1 -fold excitations:                     36
 Number of                     2 -fold excitations:                    974
 Number of                     3 -fold excitations:                  12740
 Total number of configurations:                  13751
 Calculation of coupling coefficients...
 Length of intermediate file (Mbytes):       0.9
 
 ======================================================================
 
 Spin case  1   Alpha:  0   Beta:  4
 Number of excitations:                    190
 Spin case  2   Alpha:  1   Beta:  3
 Number of excitations:                  14816
 Spin case  3   Alpha:  2   Beta:  2
 Number of excitations:                  54792
 Number of                     4 -fold excitations:                  84804
 
 Memory requirements /Mbyte/: 
               Minimal        Optimal
 Real*8:          0.7056         0.7056
 Integer:         0.4228
 Total:           1.1283         1.1283
 

 ************************ 2019-01-13 18:35:23 *************************
 Executing mrcc...

 **********************************************************************
 CCSDT(Q) calculation                                                   
 
 
 OpenMP parallel version is running.
 Number of CPUs:   1
 Allocation of    0.7 Mbytes of memory...
 Number of spinorbitals:  34
 Number of alpha electrons:  4
 Number of beta  electrons:  4
 Spin multiplicity: 1
 z-component of spin:  0.0
 Spatial symmetry: 1
 Convergence criterion:  1.0E-06
 Construction of occupation graphs...
 Number of 0-fold excitations: 1
 Number of 1-fold excitations: 36
 Number of 2-fold excitations: 974
 Number of 3-fold excitations: 12740
 Total number of determinants: 13751
 Calculation of coupling coefficients...
 Initial cluster amplitudes are generated.
 Length of intermediate file (Mbytes):       0.9
 Reading integral list from unit 55...
 Sorting integrals...
 Energy of reference determinant [au]:     -76.008350422378
 Calculation of MP denominators...
 
 Starting CC iteration...
 ======================================================================
 Norm of residual vector:     0.86650443
 CPU time [min]:     0.002                   Wall time [min]:     0.002
 
 Iteration  1  CC energy:   -76.19379235  Energy decrease:   0.18544193
 ======================================================================
 Norm of residual vector:     0.14338700
 CPU time [min]:     0.003                   Wall time [min]:     0.003
 
 Iteration  2  CC energy:   -76.19861841  Energy decrease:   0.00482606
 ======================================================================
 Norm of residual vector:     0.04155471
 CPU time [min]:     0.004                   Wall time [min]:     0.004
 
 Iteration  3  CC energy:   -76.20407725  Energy decrease:   0.00545885
 ======================================================================
 Norm of residual vector:     0.01172388
 CPU time [min]:     0.006                   Wall time [min]:     0.006
 
 Iteration  4  CC energy:   -76.20491863  Energy decrease:   0.00084138
 ======================================================================
 Norm of residual vector:     0.00345883
 CPU time [min]:     0.007                   Wall time [min]:     0.007
 
 Iteration  5  CC energy:   -76.20494533  Energy decrease:   0.00002670
 ======================================================================
 Norm of residual vector:     0.00087627
 CPU time [min]:     0.008                   Wall time [min]:     0.008
 
 Iteration  6  CC energy:   -76.20494103  Energy decrease:   0.00000430
 ======================================================================
 Norm of residual vector:     0.00030024
 CPU time [min]:     0.010                   Wall time [min]:     0.010
 
 Iteration  7  CC energy:   -76.20494007  Energy decrease:   0.00000096
 ======================================================================
 Norm of residual vector:     0.00009843
 CPU time [min]:     0.011                   Wall time [min]:     0.011
 
 Iteration  8  CC energy:   -76.20493930  Energy decrease:   0.00000077
 ======================================================================
 Norm of residual vector:     0.00002680
 CPU time [min]:     0.012                   Wall time [min]:     0.012
 
 Iteration  9  CC energy:   -76.20493947  Energy decrease:   0.00000017
 ======================================================================
 Norm of residual vector:     0.00000771
 CPU time [min]:     0.014                   Wall time [min]:     0.014
 
 Iteration 10  CC energy:   -76.20493943  Energy decrease:   0.00000004
 ======================================================================
 Norm of residual vector:     0.00000214
 CPU time [min]:     0.015                   Wall time [min]:     0.015
 
 Iteration 11  CC energy:   -76.20493941  Energy decrease:   0.00000002
 ======================================================================
 Norm of residual vector:     0.00000067
 CPU time [min]:     0.017                   Wall time [min]:     0.017
 
 Iteration 12  CC energy:   -76.20493941  Energy decrease:   2.1662E-09
 ======================================================================
 
 Iteration has converged in 12 steps.
 
 Final results:
 Total CCSDT energy [au]:            -76.204939414106
 Total MP2 energy [au]:              -76.193792350116
 
 Perturbative corrections are calculated...
 ======================================================================
 Spin case  1   Alpha:  0   Beta:  4
 Number of excitations:        190
 CPU time [min]:     0.017                   Wall time [min]:     0.017
 ======================================================================
 Spin case  2   Alpha:  1   Beta:  3
 Number of excitations:        14816
 CPU time [min]:     0.023                   Wall time [min]:     0.023
 ======================================================================
 Spin case  3   Alpha:  2   Beta:  2
 Number of excitations:        54792
 CPU time [min]:     0.034                   Wall time [min]:     0.034
 ======================================================================
 Number of 4-fold excitations: 84804
 CPU time [min]:     0.034                   Wall time [min]:     0.034
 
 Total CCSDT[Q] energy [au]:         -76.205492064990
 Total CCSDT(Q) energy [au]:         -76.205511088676
 
 ************************ 2019-01-13 18:35:25 *************************
                      Normal termination of mrcc.
 **********************************************************************
